atm_challenge.md

```diff
@@ -0,0 +1,49 @@
+# ATM challenge
+
+## Introduction
+
+This challenge is to write a software for an ATM, where a few rules, must be applied in order to
+withdraw money.
+
+Source: https://dojopuzzles.com/problems/caixa-eletronico/
+
+> PS the source is in portuguese.
+
+## Exercise
+
+Develop a software that simulates a withdraw where the the user inputs the money amount and the
+software outputs is the money bills the user can take from the machine, following the rules:
```

Comment:
Develop a software that simulates a withdrawal where the user inputs the money amount and the software outputs the money bills the user will receive from the machine, following the rules: